### ISSUE TITLE
Icon request: kin

### ISSUE THREAD

**Issue by doc-l2** [FortAwesome/Font-Awesome#16603]
**What can it be used for?**
Kin is a new way to engage, grow, and monetize your digital community. Kin enables the free and instant transfer of value between users using the Kin cryptocurrency. Currently, Kin has around 4 million monthly spenders and 6 million monthly earners and is integrated in for example Rave, Tapatalk, MadLipz and dozens more.

Including it in font-awesome would make it much easier to add Kin to apps and websites.

**Example image (optional)**
See https://github.com/kinecosystem/public-assets
Example image:
![image](https://user-images.githubusercontent.com/4309740/81471107-24460000-91ef-11ea-9dd8-4dab74cc2876.png)

**Icon request checklist**
- [x] This is a single icon or matched pair (Ex: `lock` / `unlock`)
- [x] The title starts with "Icon request: " and is followed by the requested icon name (Ex: `Icon request: magical-unicorn`)
- [x] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate
- [x] I have included at least one example image if it could be helpful (optional)
- [x] The request is for a concrete object (it's harder to make an icon to represent happiness, it's easier to make a smiley face. ☺)

**Comment by Polype01** [comment]
Nice initiative Luc! Goe bezig!

**Comment by santos1986** [comment]
🔥🔥🔥 Lekker Luc! 🔥🔥

**Comment by scara89** [comment]
🙌🙌

**Comment by tagliala** [comment]
Hi guys,

please use the upvote reaction 👍 and avoid comments

**Comment by tandembikes** [comment]
Let’s go kin

**Comment by cryptocirque** [comment]
Cool man

**Comment by cjmcgeehan** [comment]
Absolutely

**Comment by tagliala** [comment]
Hi @l-hendriks 

I think this request has been shared somewhere, asking for upvotes. If you know where, could you please specify to use the 👍  reaction instead of commenting?

Comments generate unwanted notifications and spam all the users following this request

Thanks!

**Comment by l-hendriks** [comment]
Yes I shared it on the reddit channel. I specifically added to not add a comment but to leave the comment on Reddit if they want to!

Just a small question in this case: I noticed that the leaderboard is number of upvotes plus number of comments. So if you want to discourage commenting, maybe remove the number of comments from the leaderboard?

**Comment by tagliala** [comment]
> I specifically added to not add a comment but to leave the comment on Reddit if they want to!

Thanks!

> So if you want to discourage commenting, maybe remove the number of comments from the leaderboard?

Our contributing guidelines have changed in the years, we had to use +1 comments to upvote requests before GitHub implemented reactions. That's why we are using both but this should change in the future

I'm personally checking that +1s comments are just a minority in these days and do not have huge impact on the leaderboard

**Comment by sensibleworld** [comment]
So... our backlog of brand requests has reached a point where even we can't pretend we're keeping up. Some of these issues were opened before I had gray hair. And I've had gray hair for a LONG time. On top of that, treating requests as issues has always made it way too hard to see the forest for the trees.

To fix this, we are now managing all brand requests in **GitHub Discussions**. No more +1s or thumbs-ups needed. There's an upvote system built-in.

We are closing existing brand requests as part of this transition, but **we are not saying no** to your request.

**If this brand request is still relevant for you, please re-open it in Discussions** (and feel free to include a link back to this issue):
-> https://github.com/FortAwesome/Font-Awesome/discussions/new?category=brand-requests

And listen. We know this is annoying. If you need to vent, here are some pre-written responses you can use when opening the new discussion:

* Cool cool cool. No worry at all. I love re-requesting a brand that's redesigned three times while this issue was open.
* If an issue closes after seven years and nobody responds, did it ever really exist?
* I see you have chosen violence today. Re-opening per instructions.
* What's next? I have to file a TPS report, too?
* I am filled with the fury of a thousand suns. Why do I have to re-open the discussion? Couldn't you have just bulk converted all of the issues to discussions? What? You couldn't? What do you mean Github removed that feature earlier this year? You'd have to manually click the "convert to discussion" button on thousands of issues, while also ensuring it gets added to the correct category? That's a bunch of bull-phooey.
* No worries.
* This brand has gone the way of the dodo.
